<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmClient</class>
 <widget class="QWidget" name="FrmClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MST - Client</string>
  </property>
  <widget class="QLabel" name="lblInfoServer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>341</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>lblInfoServer</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>381</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lblBandwidth">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>211</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>lblBandwidth</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblBandwidthvalue">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>lblBandwidthvalue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>mbit/s</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>390</y>
     <width>561</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>491</width>
     <height>130</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="lblResolution">
          <property name="text">
           <string>lblResolution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbResolution"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="lblFps">
          <property name="text">
           <string>lblFps</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbFps"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="lblBuffer">
          <property name="text">
           <string>lblBuffer</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbBuffer"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="lblVideo">
          <property name="text">
           <string>lblVideo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbVideo"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>238</width>
        <height>125</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="lblColorScale">
          <property name="text">
           <string>lblColorScale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbColorScale"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="lblEncoding">
          <property name="text">
           <string>lblEncoding</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbEncoding"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <widget class="QLabel" name="lblAudio">
          <property name="text">
           <string>lblAudio</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbAudio"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <widget class="QLabel" name="lblControls">
          <property name="text">
           <string>lblControls</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbControls"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>531</width>
     <height>29</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QPushButton" name="btnBack">
      <property name="text">
       <string>btnBack</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_15">
      <item>
       <widget class="QPushButton" name="btnLoadConfig">
        <property name="text">
         <string>btnLoadConfig</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnSaveConfig">
        <property name="text">
         <string>btnSaveConfig</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnConnect">
      <property name="text">
       <string>btnConnect</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblConfig">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>413</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>lblConfig</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>284</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="lblIpServer">
        <property name="text">
         <string>lblIpServer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPort">
        <property name="text">
         <string>lblPort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblPassword">
        <property name="text">
         <string>lblPassword</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="txtIpServer">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPort"/>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPassword"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="lblIpClient">
      <property name="text">
       <string>lblIpClient</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lblIpClient2">
      <property name="text">
       <string>lblIpClient2</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
